src/Cardano/Shell/Types.hs

```diff
@@ -1,6 +1,38 @@
-{-# LANGUAGE Rank2Types #-}
-
-module Cardano.Shell.Types where
+module Cardano.Shell.Types
```

Comment:
Please do move these types in a separate module, as I mentioned.
We don't want to pollute the top-level namespace with things that are not its responsibility.

Reply:
Oh I thought that was just the mainnetConfiguration and devConfiguration.
Should I move everything below "-- Cardano Configuration Data Structures" into ConfigurationTypes.hs ?

Reply:
You can try the folder/package `Constants` - so `Constants.Types`.

Reply:
Changes implemented.